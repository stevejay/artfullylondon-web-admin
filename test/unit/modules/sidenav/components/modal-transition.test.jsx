import React from 'react'
import { shallow } from 'enzyme'

import ModalTransition from '_src/modules/sidenav/components/modal-transition'

it('should render correctly', () => {
  const wrapper = shallow(<ModalTransition><div id='child' /></ModalTransition>)
  expect(wrapper).toMatchSnapshot()
})
