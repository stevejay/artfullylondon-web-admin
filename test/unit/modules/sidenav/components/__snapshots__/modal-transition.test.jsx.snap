// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<BasicTransition
  appear={false}
  className="src-modules-sidenav-components-modal-transition__modal-transition"
  in={false}
  mountOnEnter={false}
  timeoutMs={200}
  unmountOnExit={false}
>
  <div
    id="child"
  />
</BasicTransition>
`;
